.gitignore

```diff
@@ -30,6 +30,7 @@ local.properties
 *.apk
 *.iml
 *.jks
+android/gradle.properties
```

Comment:
In gradle, it will look at your home directory (ex `~/.gradle/gradle.properties`) for the value and take precedence over the version controlled `android/gradle.properties`.  This allows you to permanently add the key to your home directory `~/.gradle/gradle.properties` which will avoid risk of checking a key into version control.